src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/WinformsDefaultFontUpdater.cs

```diff
@@ -23,7 +23,7 @@ public class WinformsDefaultFontUpdater : IUpdater<IProject>
 
         public string Title => "Default Font API Alert";
 
-        public string Description => "Default font in Windows Forms has been changed from Microsoft Sans Serif to Seg Segoe UI, in order to change the default font use the API - Application.SetDefaultFont(Font font). For more details see here - https://devblogs.microsoft.com/dotnet/whats-new-in-windows-forms-in-net-6-0-preview-5/#application-wide-default-font";
+        public string Description => Resources.DefFontMessage;
```

Comment:
nit: can you spell out the full name? i.e. `DefaultFontMessage`. no need to shorten things